src/System.Memory/ref/System.Memory.cs

```diff
@@ -112,6 +112,24 @@ public static partial class MemoryExtensions
 }
 namespace System.Buffers
 {
+    public sealed partial class ArrayBufferWriter : System.Buffers.IBufferWriter<byte>, System.IDisposable
+    {
+        public ArrayBufferWriter() { }
+        public ArrayBufferWriter(int initialCapacity) { }
+        public int BytesAvailable { get { throw null; } }
+        public int BytesWritten { get { throw null; } }
+        public int Capacity { get { throw null; } }
+        public System.ReadOnlyMemory<byte> OutputAsMemory { get { throw null; } }
+        public System.ReadOnlySpan<byte> OutputAsSpan { get { throw null; } }
+        public long TotalBytesWritten { get { throw null; } }
+        public void CopyToAndReset(System.IO.Stream stream) { }
+        public System.Threading.Tasks.Task CopyToAndResetAsync(System.IO.Stream stream, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public void Dispose() { }
+        public void Reset() { }
+        void System.Buffers.IBufferWriter<System.Byte>.Advance(int count) { }
```

Comment:
The terminology used is confusing to me. Why do we really about bytes written when you never write anything via these APIs, rather you advance. Shouldn't it then be BytesAdvanced?